### PR TITLE
Setup base Github Actions support

### LINKED ISSUE #2: GitHub Actions Support
Wrong repo, I know, but any plans for a GitHub Actions Discord Hook?

### PR DESCRIPTION
**Pull request by lcombs15**
Closes #2 

Passing Build:
image

Failing Build:
image

### CODE REVIEW

**Review by tiemonl — APPROVED** [review]

**Review by tiemonl — APPROVED** [review]

File: send.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# This original source of this code: https://github.com/DiscordHooks/travis-ci-discord-webhook
+# The same functionality from TravisCI is needed for Github Actions
+#
+# For info on the GITHUB prefixed variables, visit:
+# https://help.github.com/en/articles/virtual-environments-for-github-actions#environment-variables
+if [ -z "$2" ]; then
+  echo -e "WARNING!!\nYou need to pass the WEBHOOK_URL environment variable as the second argument to this script.\nFor details & guide, visit: https://github.com/DiscordHooks/travis-ci-discord-webhook" && exit
```
Comment (iamtraction) [inline send.sh]:
```suggestion
  echo -e "WARNING!!\nYou need to pass the WEBHOOK_URL environment variable as the second argument to this script.\nFor details & guide, visit: https://github.com/DiscordHooks/github-actions-discord-webhook" && exit
```

File: send.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+fi
+
+echo -e "[Webhook]: Sending webhook to Discord...\\n";
+
+AVATAR="https://avatars0.githubusercontent.com/u/44036562?s=200"
```
Comment (iamtraction) [inline send.sh]:
```suggestion
AVATAR="https://github.com/actions.png"
```

**Review by iamtraction — CHANGES_REQUESTED** [review]
Just a few changes. Other than that, it looks good to me. 👌

**Review by iamtraction — APPROVED** [review]
LGTM 👌 
And you've been invited to DiscordHooks org. Welcome!

**Pull request merged**